test/unit/tool_util/test_tool_linters.py

```diff
@@ -174,6 +175,36 @@
 </tool>
 """
 
+# check that linter does complain about tests wo assumptions
+TESTS_WO_EXPECTATIONS = """
+<tool>
+    <tests>
+        <test>
+        </test>
+    </tests>
+</tool>
+"""
+
+TESTS_PARAM = """
+<tool>
+    <inputs>
+        <param name="existent_test_name"/>
```

Comment:
Maybe test `--argument` here?

```suggestion
        <param argument="--existent-test-name"/>
```